lxd/apparmor/archive.go

```diff
@@ -108,16 +115,57 @@ func archiveProfile(s *state.State, outputPath string, allowedCommandPaths []str
 		outputPathFull = outputPath // Use requested path if cannot resolve it.
 	}
 
-	backupsPath := s.BackupsStoragePath("")
-	backupsPathFull, err := filepath.EvalSymlinks(backupsPath)
-	if err == nil {
-		backupsPath = backupsPathFull
+	// Add all paths configured as daemon storage or project storage.
+	// We store the paths in a map[string]bool to ensure uniqueness.
+	daemonStorageVolumePaths := make(map[config.DaemonStorageType]map[string]bool)
+	daemonStorageVolumePaths["images"] = make(map[string]bool)
```

Comment:
Not a block, but please can you follow up to change the key type of this map to StorageType